### PR TITLE
provide default implementation for Message.getAutoTrack()

### PR DESCRIPTION
**Pull request by rymorale**
## Description
- Provide a default implementation for getAutoTrack() in the Message interface as it was previously present in the now internal InternalMessage class.
- also fix logging bug in MessagingExtension

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: code/messaging/src/main/java/com/adobe/marketing/mobile/Message.java [hunk 1]
```diff
@@ -79,4 +81,13 @@ public interface Message {
...
      */
     void setAutoTrack(boolean enabled);
+
+    /**
+     * Gets the {@link Message}'s auto tracking preference.
+     *
+     * @return {@code boolean} containing the auto tracking preference.
+     */
+    default boolean getAutoTrack() {
+        return autoTrack;
```
Comment (praveek) [inline code/messaging/src/main/java/com/adobe/marketing/mobile/Message.java:91]: Is true the default value? If so, can you remove the `autoTrack` variable and just return "true"

Comment (rymorale) [reply code/messaging/src/main/java/com/adobe/marketing/mobile/Message.java:91]: makes sense as true is default.

File: code/messaging/src/main/java/com/adobe/marketing/mobile/Message.java [hunk 2]
```diff
@@ -79,4 +79,13 @@ public interface Message {
      * @param enabled {@code boolean} if true, tracking is done automatically for {@code Message} show, dismiss, and triggered events.
      */
     void setAutoTrack(boolean enabled);
```
Comment (swarna04) [inline code/messaging/src/main/java/com/adobe/marketing/mobile/Message.java:81]: Should this be `final` boolean enabled?

**Review by swarna04 — APPROVED** [review]

File: code/messaging/src/main/java/com/adobe/marketing/mobile/Message.java (hunk 2 above)
Comment (rymorale) [reply code/messaging/src/main/java/com/adobe/marketing/mobile/Message.java:81]: i'll update this

**Pull request merged**